src/resources/snippets.ts

```diff
@@ -226,5 +226,5 @@ export default [
     "description": "Makes the song progress and volume bar thicker",
     "code": ".x-progressBar-progressBarBg { height: 100% !important; } .x-progressBar-sliderArea { height: 100% !important; } .x-progressBar-fillColor { height: 100% !important; }",
     "preview": "resources/assets/snippets/thicker-bars.png",
-  },
+  }
```

Comment:
Trailing coma for lint check
```suggestion
  },
```

Reply:
a space after code too line 77